tests/javascript_tests/test_color_view_controller.js

```diff
@@ -126,7 +126,7 @@ requirejs([
 
     test("Test ColorViewController.getColorList exceptions", function(){
       var five;
-      five = [0, 1, 2, 3, 4];
+      five = ['0', 'string1', 'string2', 'string3', 'string4'];
```

Comment:
I'm skeptical that we are testing all the separate functions by just adding a new test case 😆 can you add the appropriate test cases?